### ISSUE TITLE
Mandatory fields in customer details

### ISSUE THREAD

**Issue by biospirt** [webmasterlv/swedbank-spp#2]
Tnx for library. It is nice and easy to use. 
I just did not pass card payment tests because first_name and last_name turned out to be mandatory. 
Can you please correct documentation for this?

**Comment by webmasterlv** [comment]
Hey, thank your for report.
I tried to make a payment without first name and last name, but it opened payment form without problems. This is strange.
Anyway, maybe you should try anonymous payment option i just implemented, i got report some time ago that this was possible.

**Comment by biospirt** [comment]
Yes- payment was working fine, but bank dod not accept test payments until i filled up those fields. They said those fields are mandatory and sent me error they saw from their side.
